### ISSUE TITLE
Add support for built-in function 'tostring'

### ISSUE THREAD

**Issue by kenjones-cisco** [mikefarah/yq#72]
Add support for built-in [function 'tostring'](https://stedolan.github.io/jq/manual/#example41)

```
The tostring function prints its input as a string. Strings are left unchanged, and all other values are JSON-encoded.
```

**Comment by b-dean** [comment]
@mikefarah, this says it was completed ages ago, but I cannot figure out how to do it. Nothing seems to know about `tostring` and it's not documented.

```sh
yq eval --null-input '1 | tostring'
```

produces:
```
Error: 1:5: invalid input text "tostring"
```

If this is complete, how are we supposed to use it?

**Comment by kro-cat** [comment]
For others, the `tostring` operator doesn't exist in yq. However, for future googlers, I found the `@yaml` encode filter ( or others like it, see [encode-decode](https://mikefarah.gitbook.io/yq/operators/encode-decode) form the yq docs ) seems to work for my specific use case:

Expected result:
```yaml
items:
  - object: |
      array:
        - |
          a: foo
          b: bar
```

yq filter:

```jq
{
  "items": [
    {
      "object": ({
        "array": [
          ({
            "a": (.path.to.foo),
            "b": "bar"
          } | @yaml)
        ]
      } | @yaml)
    }
  ]
}
```

**Comment by mikefarah** [comment]
Sorry, this was closed with a whole bunch of other cards that didn't have any traction - will reopen.

**Comment by mikefarah** [comment]
Will be fixed in the next release

**Comment by mikefarah** [comment]
Fixed in v4.43.1 :)
